fe/fe-core/src/main/java/com/starrocks/statistic/StatisticExecutor.java

```diff
@@ -219,9 +219,11 @@ public void collectStatistics(StatisticsCollectJob tcj) {
 
             GlobalStateMgr.getCurrentStatisticStorage().expireColumnStatistics(table, columns);
         } catch (Exception e) {
-            analyzeJob.setWorkTime(LocalDateTime.now());
-            analyzeJob.setReason(e.getMessage());
-            GlobalStateMgr.getCurrentAnalyzeMgr().updateAnalyzeJobWithLog(analyzeJob);
+            if (analyzeJob.getId() != -1) {
```

Comment:
Please add comments to explain why -1 is exception.

Reply:
Add comment done.

// If the job id is equal to -1, it represents the automatic full statistics collection of the new version.
// Automatic full statistics collection is an implicit task
// that is not recorded in the analyze job list, nor does it update the status of the analyze job.